aws/tags.go

```diff
@@ -111,15 +111,16 @@ func SetTagsDiff(_ context.Context, diff *schema.ResourceDiff, meta interface{})
 
 	allTags := defaultTagsConfig.MergeTags(resourceTags).IgnoreConfig(ignoreTagsConfig)
```

Comment:
Unrelated to this issue I'm sure, but does this line also require `.IgnoreAws()`?

Reply:
I'm guessing there might be strange behavior if the `CustomizeDiff` function is removing `aws:` keys, but they are actually configured. Might be best to create a tracking issue for what happens in such a scenario. My guess would be that it will already trigger a perpetual difference, which is likely the desirable behavior.

Reply:
OK, `TestAccAWSS3Bucket_tagsWithSystemTags` is already testing `aws:` tag functionality.
I'll create a variant of that with `default_tags` and see what happens...

Reply:
tracking issue created here : https://github.com/hashicorp/terraform-provider-aws/issues/19019 (feel free to add more context :) , close if it doesn't prove applicable )